legislativas/20220130_legislativas/README.md

```diff
@@ -24,7 +24,7 @@ A disponibilização dos programas num formato standard permitirá também que s
 | Partido da Terra  |  |  | |
 | Partido Popular Monárquico | | | |
 | Partido Social Democrata |  |  | |
-| Partido Socialista | |  | |
+| Partido Socialista |[Linhas Gerais](https://ps.pt/wp-content/uploads/2021/09/Linhas-Gerais-do-Programa-Eleitoral-2022_5jan.pdf) [12 Prioridades](https://ps.pt/wp-content/uploads/2021/09/12-prioridades-para-os-proximos-4-anos_5jan-1.pdf) |  | |
```

Comment:
Não é bem o programa eleitoral. Sugeria esperarmos mais algum tempo para ver se o PS disponibiliza o programa em si

Reply:
@tiago-sousa confirma o estado em que tens a conversão, se for programa do PS. É que já ia lançado e converti a 1a metade, mas como não fiz assignee no repo, mas óbvio que se já tiveres a fazer fica o teu.

Reply:
Podes avançar @joaosaro, nenhum dos documentos que o @tiago-sousa colocou era o programa eleitoral e o que tu colocaste sim. IMO os documentos deste PR seriam usados à falta de melhor :)

Dito isto, por mim pode-se fechar este PR